### ISSUE TITLE
Improve hover labels

### ISSUE THREAD

**Issue by jethror1** [ewels/MegaQC#328]
**Is your feature request related to a problem? Please describe.**

Improve labels displaying on plots when hovering over data points.

**Describe the solution you would like**

Do you do any formatting of the hover labels for plotly? It could be nice if they would show the run a sample came from, and also if the value was on a separate line to the date for clarity. I think something like this might be nice:

```
#Current:
(Oct 19, 2021. 17:30, 67)
MB1274-227742-RNA-egg6_S26_L002_R2

#Suggested
Value: 67
Date: Oct 19, 2021. 17:30
Sample: MB1274-227742-RNA-egg6_S26_L002_R2
Report: run-xxx
```
For the run information I'm not sure what is appropriate / generalisable to others, for us `config_analysis_dir` is the name of sequencing run. This would be useful to see with larger numbers of samples on a plot to easily identify which run they came from.

Plotly makes this hover text reasonably easy to configure.

**Additional context**

![Screenshot from 2021-12-14 12-59-30](https://user-images.githubusercontent.com/45037268/146003800-1ed3213b-942c-4831-8637-0af7229e4557.png)

**Comment by multimeric** [comment]
Yeah so this is tricky to make generally useful. For each user the metadata has different meanings. I think it would be okay if we made it so that you can choose the metadata fields shown for labels in each plot. Then you could save these settings along with the other plot settings.

**Comment by ewels** [comment]
Could we potentially bind a click event on the plot data point to take you to a "report" page, showing metadata and samples from the MultiQC run?

**Comment by multimeric** [comment]
Probably, but that doesn't solve the at-a-glance clarity we want to provide via the tooltips exactly.
